docs/guide/acquiring_data/database.rst

```diff
@@ -753,22 +748,23 @@ with the value 'Angstrom':
 
     >>> import sunpy.database.attrs as dbattrs
     >>> print(display_entries(
-    ...     database.query(dbattrs.Tag('spring') | dbattrs.Starred(), ~dbattrs.FitsHeaderEntry('WAVEUNIT', 'Angstrom')),
+    ...     database.search(dbattrs.Tag('spring') | dbattrs.Starred(), ~dbattrs.FitsHeaderEntry('WAVEUNIT', 'Angstrom')),
     ...     ['id', 'observation_time_start', 'observation_time_end',
     ...      'instrument', 'wavemin', 'wavemax', 'tags', 'starred'], sort=True))   # doctest: +NORMALIZE_WHITESPACE +SKIP
-    id observation_time_start observation_time_end instrument wavemin wavemax tags   starred
-    -- ---------------------- -------------------- ---------- ------- ------- ----   -------
-    1  2011-03-19 10:54:00    2011-03-19 10:54:00  AIA_3      17.1    17.1    spring No
-    26 2011-05-08 00:00:00    2011-05-08 00:00:01  AIA        17.1    17.1    spring No
-    27 2011-05-08 00:00:00    2011-05-08 00:00:01  AIA        21.1    21.1    spring Yes
-    28 2011-05-08 00:00:02    2011-05-08 00:00:03  AIA        9.4     9.4     spring No
-    29 2011-05-08 00:00:03    2011-05-08 00:00:04  AIA        33.5    33.5    spring Yes
-    32 2012-08-05 00:00:02    2012-08-05 00:00:03  AIA        33.5    33.5    N/A    Yes
-    33 2012-08-05 00:00:02    2012-08-05 00:00:03  AIA        33.5    33.5    N/A    Yes
-    36 2013-08-05 00:00:02    2013-08-05 00:00:03  AIA        33.5    33.5    N/A    Yes
-    37 2013-08-05 00:00:02    2013-08-05 00:00:03  AIA        33.5    33.5    N/A    Yes
-    4  2011-03-19 10:54:00    2011-03-19 10:54:00  AIA_3      17.1    17.1    spring No
-    5  2014-04-09 06:00:12    2014-04-09 06:00:12  AIA_3      17.1    17.1    spring No
+    id observation_time_start observation_time_end ... wavemax  tags  starred
```

Comment:
Why is truncating the results? is that because it's been run in a smaller window??

Reply:
I've run every code in Jupyter QT Console more than once, 
every time the output was truncated. So may be it's because of smaller window itself.

Reply:
Could you try to make it a larger window? I don't really understand why it is truncated. What's the point to choose the fields if then we don't see these?

Reply:
This output is from pytest log I dump to a file.